src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -630,6 +677,14 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
         return false;
     }
 
+    yarp::os::Value falseValue;
+    falseValue.fromString("false");
+    if (prop.check("publishOnROS", falseValue, "Checking ROS enabled").asBool()) {
```

Comment:
Can you add this two options to the device documentation in `WholeBodyDynamicsDevice.h`?